### PR TITLE
Add blog posts for HODL

### PR DESCRIPTION
**Pull request by at2005**

### CODE REVIEW

**Comment by willzeng** [comment]
Hi @at2005 Thanks for posting! Is this ready for review?

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Thanks @at2005! Excellent job. Some minor technicalities:
- Can you please generate the html page, using the pandoc commands reported in the [readme](https://github.com/unitaryfund/unitary-fund#writing-a-blog-post).
Please add a link to the html page to the blog index, blog.html?
- Also, please have a double check at the html file locally to check that pandoc converted it correctly, e.g., https://github.com/unitaryfund/unitary-fund/issues/243.
- What md file needs to be included, there are two now
- there is an image at the top that I think is not supposed to be there?
- I'm not familiar with the way images are included: can they included in the website for more reliability in the future.

**Comment by at2005** [comment]
Hi, I've updated the files accordingly :) There is now one HTML file (hodl.html), and have edited it to format it properly and added the link to blog.html.

File: posts/_markdown/qiskit_hodl.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+[![Ayush Tambde](https://miro.medium.com/fit/c/96/96/1*HzoP0H_p6w03dx6YVNfwZQ.jpeg)
+
+](https://medium.com/@atamb?source=post_page-----377859287861--------------------------------)[Ayush Tambde](https://medium.com/@atamb?source=post_page-----377859287861--------------------------------)Follow
+
+Jun 28
```
Comment (nathanshammah) [inline posts/_markdown/qiskit_hodl.md:5]:
```suggestion
2022-08-19
```

File: posts/hodl.html [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+
+                            <right>
+                                <h2 class="leading-arrow"></h2>
+                                <h3>An
+                                Introduction to Creating Quantum Oracles
+                                with HODL</h3>
+                            </right>
+                            <div>
+                                <p>Jun 28</p>
```
Comment (nathanshammah) [inline posts/hodl.html:78]:
```suggestion
                                <p>2022-08-19</p>
```

File: blog.html [hunk 3]
```diff
@@ -71,6 +71,27 @@
               </p>
               <section class="light">
                 <ul style="list-style-type: none">
+                  <li class="leading-block">
+                    <span class="blog-post-date">7 August, 2022</span>
```
Comment (nathanshammah) [inline blog.html:75]:
```suggestion
                    <span class="blog-post-date">19 August, 2022</span>
```

**Review by nathanshammah — APPROVED** [review]

**Pull request merged**